frontend/src/pages/Landing/__tests__/index.js

```diff
@@ -281,7 +274,7 @@ describe('Landing Page sorting', () => {
       { count: 2, rows: activityReportsSorted },
     );
 
-    fireEvent.click(statusColumnHeader);
+    await act(async () => fireEvent.click(statusColumnHeader));
```

Comment:
I wonder if you remove the `useState` and `useEffect` from the `TooltipWithEllipsis` component you will also be able to remove the `act` wrapping here and have the tests run without the "act" warning.